README.md

```diff
@@ -2,113 +2,40 @@
 
 Internal Knowledge Database for Zenika members - https://faq.zenika.com
 
-## Getting Started
+## What is FAQ?
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+**FAQ** is an **internal knowledge database** for Zenika members. It aims to be the **"single source of truth"** for most of Zenika-oriented information: "How does the variable part of the salary works?", "Can the client ask me to stay late at work?", ...
```

Comment:
There should be not "s" at the end of "works" because that mark borne by "does".